opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -133,6 +133,7 @@ def __init__(
         links: Sequence[trace_api.Link] = (),
         kind: trace_api.SpanKind = trace_api.SpanKind.INTERNAL,
         span_processor: SpanProcessor = SpanProcessor(),
+        creator_info: "InstrumentationInfo" = None,
```

Comment:
Any reason not to call this argument `instrumentation_info`?

Reply:
Because it is the instrumentation_info that identifies the creator of this span, so I thought `creator_info` is a more specific name.

Reply:
I renamed it in 725bb161f9c22285893b6756230e7bad2fdbc6b9.